sdk/go/arvadosclient/arvadosclient.go

```diff
@@ -149,30 +167,36 @@ func (this ArvadosClient) CallRaw(method string, resource string, uuid string, a
 	}
 
 	defer resp.Body.Close()
-	errorText := fmt.Sprintf("API response: %s", resp.Status)
+	return nil, NewRemoteApiServerError(this.ApiServer, resp)
+}
+
+func NewRemoteApiServerError(ServerAddress string, resp *http.Response) RemoteApiServerError {
```

Comment:
Makes sense. I changed the names to `APIServerError` and `newAPIServerError()`. I left the `Server` string in the error name, because I worry that users will misread `APIError` as `SDKError` and incorrectly conclude (as I did) that this error occurred locally on their machine.